test/cloudformation/iam_cloudformation.yaml

```diff
@@ -39,6 +39,112 @@ Resources:
     Type: AWS::IAM::ManagedPolicy
     Properties:
       ManagedPolicyName: GithubActionsPolicy
+      PolicyDocument:
```

Comment:
What was the need for moving this here? The current IAM policy gets all of the permissions from the permission boundary already